src/logic/addressBook/model/addressBook.ts

```diff
@@ -1,22 +1,22 @@
-import { ETHEREUM_NETWORK } from 'src/config/networks/network.d'
-import { getNetworkId } from 'src/config'
+import { _getChainId } from 'src/config'
+import { ChainId } from 'src/config/chain.d'
 
 export const ADDRESS_BOOK_DEFAULT_NAME = 'UNKNOWN'
 
 export type AddressBookEntry = {
   address: string // the contact address
   name: string // human-readable name
-  chainId: ETHEREUM_NETWORK // see https://chainid.network
+  chainId: ChainId // see https://chainid.network
 }
 
 export const makeAddressBookEntry = ({
   address,
   name,
-  chainId = getNetworkId(),
+  chainId = _getChainId(),
```

Comment:
A brief grep showed that `makeAddressBookEntry` is always called from places that can already access Redux.